middle_end/flambda2/to_cmm/to_cmm_expr.ml

```diff
@@ -553,7 +558,8 @@ and let_cont_not_inlined env res k handler body =
       let_cont_exn_handler env res k body vars handler free_vars_of_handler
         ~catch_id arity
     else
-      let dbg = Debuginfo.none (* CR mshinwell: fix debuginfo *) in
+      (* CR mshinwell: fix debuginfo *)
+      let dbg = Env.add_inlined_debuginfo env Debuginfo.none in
```

Comment:
If the debuginfo was "none", we probably shouldn't add the inlined debuginfo - might be confusing?

Reply:
Indeed, I reverted back to using `Debuginfo.non` with a CR to remember to apply `Env.add_inlined_debuginfo` when a proper debuginfo is available.